doc-content/enterprise-only/installation/clustering-bc-configure-domain-proc.adoc

```diff
@@ -1,15 +1,15 @@
 [id='clustering-bc-configure-domain-proc']
 = Configuring a {EAP} {EAP_VERSION} domain-mode cluster for {CENTRAL}
-Configure the {CENTRAL} cluster in domain mode with Elasticsearch and the JMS broker resource adapter integrated with {EAP}. By using domain mode with a cluster, you can centrally manage and publish configurations for your servers.
+Configure the {CENTRAL} cluster in domain mode with {DATAGRID} and the JMS broker resource adapter integrated with {EAP}. By using domain mode with a cluster, you can centrally manage and publish configurations for your servers.
 
```

Comment:
you can centrally manage and publish configurations of* your servers.

Reply:
Done